pages/docs/installation/installation-special-systems.md

```diff
@@ -794,3 +794,35 @@ export PRECICE_BOOST_ROOT=$HOME/software/boost_1_53_0
 * configure ALYA with `-L[PathToPreCICE]/build/last -lprecice -lstdc++ -lrt`
 * for running: also put `module load intel` in your jobscript
 * use `network="ib0"` for sockets communication beyond one node
+
+### Max Planck Computing and Data Facility (MPCDF) Cobra cluster
+
+#### Installing dependencies
+
+##### Eigen3
+
+See the [Eigen dependency section](https://precice.org/installation-source-dependencies.html#eigen).
```

Comment:
You could ask the admins to provide a module for this especially as it is an extremely common library.

Reply:
I asked and there is no requirement for Eigen other than preCICE so it will not be added as a module